### PR TITLE
Add confirmation dialog on account remove cli

### PR DESCRIPTION
**Pull request by jhs7jhs**
#612

### CODE REVIEW

File: core/src/account_provider.rs [hunk 1]
```diff
@@ -124,8 +124,8 @@ impl AccountProvider {
         Ok((address, public))
     }
 
-    pub fn remove_account(&self, address: Address, password: &Password) -> Result<(), SignError> {
-        Ok(self.keystore.write().remove_account_with_password(&address, password)?)
```
Comment (sgkim126) [inline core/src/account_provider.rs]: It seems that `remove_account_with_password` is not used anymore.

File: codechain/subcommand/account_command.rs [hunk 2]
```diff
@@ -121,6 +124,16 @@ fn prompt_password(prompt: &str) -> Password {
     rpassword::prompt_password_stdout(prompt).map(Password::from).unwrap()
 }
 
+fn confirmation_dialog(prompt: &str) -> Result<bool, String> {
+    println!("Type \"{}\" to confirm: ", prompt);
+    let mut input = String::new();
+    match std::io::stdin().read_line(&mut input) {
```
Comment (sgkim126) [inline codechain/subcommand/account_command.rs]:
```rust
std::io::stdin().read_line(&mut input).map_err(|e| format!("Failed to read line: {}", e))?;
```

File: core/src/account_provider.rs (hunk 1 above)
Comment (jhs7jhs) [reply core/src/account_provider.rs]: I think it can be deleted.

Comment (jhs7jhs) [reply core/src/account_provider.rs]: But multistore can have the same account with different password. Is this intended? `remove_account` will delete all of them.

Comment (jhs7jhs) [reply core/src/account_provider.rs]: I found out that `KeyStore` blocks the multi-account situation on `KeyMultistore`. I think it's okay to delete it.

File: codechain/subcommand/account_command.rs [hunk 3]
```diff
@@ -94,9 +94,12 @@ fn import_raw(ap: &AccountProvider, network_id: NetworkId, raw_key: &str) -> Res
 
 fn remove(ap: &AccountProvider, address: &str) -> Result<(), String> {
     let address = PlatformAddress::from_str(address).map_err(|err| err.to_string())?;
-    let password = prompt_password("Password: ");
-    let _ = ap.remove_account(address.into_address(), &password).map_err(|err| err.to_string())?;
-    println!("{} is deleted", address);
+    if confirmation_dialog("REMOVE")? {
```
[the remark this answers is not in the document]
Comment (jhs7jhs) [reply codechain/subcommand/account_command.rs]: @sgkim126 fixed

**Pull request merged**